presto-common/src/main/java/com/facebook/presto/common/Utils.java

```diff
@@ -126,4 +131,23 @@ public static void checkState(boolean test, String errorMessage)
             throw new IllegalStateException(errorMessage);
         }
     }
+
+    /**
+     * Constructs the schema name, including catalog name if applicable.
+     *
+     * @param schemaName the original schema name
+     * @return the formatted schema name (Example - @catalog_name#schema_name)
+     */
+    public static String constructSchemaName(Optional<String> catalogName, String schemaName)
+    {
+        if (catalogName.isPresent() && schemaName != null && !schemaName.equals(DEFAULT_DATABASE) && !schemaName.contains(CATALOG_DB_SEPARATOR)) {
```

Comment:
If `schemaName` can be null here, please add a `@Nullable` annotation to the method parameter